bin/conventional-changelog.js

```diff
@@ -248,7 +246,7 @@ class ConventionalChangelog {
     }
 
     changelog.entries.unshift(newEntry)
-    await writeFile(filename, JSON.stringify(changelog, null, 2), 'utf-8')
+    await writeFile(jsonFile, JSON.stringify(changelog, null, 2), 'utf-8')
```

Comment:
I'm not sure if we really care but the changelog.json entries will have PR links in them that are not linked

Reply:
I'm updating to remove them, nbd